### PR TITLE
Feature/281685 content declaration

### PR DESCRIPTION
**Pull request by alex742**

### CODE REVIEW

File: app/routes/endemics/declaration.js [hunk 1]
```diff
@@ -86,8 +87,10 @@ module.exports = [{
         throw boom.internal()
       }
 
+      console.log(application, application.organisation)
```
Comment (aldeez988) [inline app/routes/endemics/declaration.js:90]: please remove this console log

File: app/routes/endemics/declaration.js [hunk 2]
```diff
@@ -86,8 +87,10 @@ module.exports = [{
...
       return h.view('endemics/confirmation', {
```
Comment (aldeez988) [inline app/routes/endemics/declaration.js:91]: can we add this link to the configs/routes and reused from there

File: app/routes/endemics/declaration.js [hunk 3]
```diff
@@ -86,8 +87,10 @@ module.exports = [{
...
         reference: applicationReference,
+        newUser: appliedBefore.NEW_USER === application.organisation.appliedBefore,
```
Comment (aldeez988) [inline app/routes/endemics/declaration.js:93]: I can see  **newUser** property will contain boolean suggest changing it to **isNewUser** for better code readability

**Review by aldeez988 — CHANGES_REQUESTED** [review]

File: app/api-requests/business-applied-before.js [hunk 4]
```diff
@@ -1,7 +1,6 @@
 const applicationApi = require('./application-api')
 const status = require('../constants/status')
 const { appliedBefore } = require('../constants/user-types')
-const { OutstandingAgreementError, AlreadyAppliedError } = require('../exceptions')
 
 async function businessAppliedBefore (sbi) {
   const latestApplicationsForSbi = await applicationApi.getLatestApplicationsBySbi(sbi)
```
Comment (aldeez988) [inline app/api-requests/business-applied-before.js:6]: is this api call ? if api call I think we should add try and catch block for it

File: app/api-requests/business-eligible-to-apply.js [hunk 5]
```diff
@@ -24,9 +25,13 @@ function applicationForBusinessInStateToApply (latestApplicationsForSbi) {
     console.log(`${new Date().toISOString()} Business is eligible to apply : ${JSON.stringify({
         sbi: latestApplication.data.organisation.sbi
       })}`)
-  // when toggle is on
-  } else if (latestApplication.statusId === status.AGREED && latestApplication.type === 'EE') {
-    throw new AlreadyAppliedError(`Business with SBI ${latestApplication.data.organisation.sbi} already has an endemics agreement`)
+  } else if (config.endemics.enabled) {
+    if (latestApplication.statusId === status.AGREED && latestApplication.type === 'EE') {
```
Comment (aldeez988) [inline app/api-requests/business-eligible-to-apply.js:29]: What does **EE** refer to? we should add it to constant for better code readability

**Review by aldeez988 — DISMISSED** [review]

**Review by Afshin-Karamifar-Capgemini — APPROVED** [review]

**Review by aldeez988 — APPROVED** [review]

**Pull request merged**